sqlalchemy_ibmi/base.py

```diff
@@ -627,18 +611,9 @@ def post_exec(self):
 class DB2ExecutionContext(_SelectLastRowIDMixin,
                           default.DefaultExecutionContext):
     """IBM i Db2 Execution Context class"""
-
-    # TODO These methods are overridden from the default dialect and should be
-    #  implemented
-
-    def create_server_side_cursor(self):
-        pass
-
-    def result(self):
-        pass
-
-    def get_rowcount(self):
-        pass
+    # create_server_side_cursor not implemented because supports_server_side is
+    # set to False in the dialect. get_rowcount is replaced with rowcount
+    # result replaced by get_result_proxy in super
```

Comment:
I'm not sure these comments are needed in the code itself. Would be great to have in the commit message, though. Would be great to explain why these stub functions were added in the first place and why they are being removed now.